src/test/kotlin/racingcar/domain/CarTest.kt

```diff
@@ -1,23 +1,30 @@
 package racingcar.domain
 
+import io.kotest.assertions.throwables.shouldThrow
 import io.kotest.core.spec.style.DescribeSpec
 import io.kotest.matchers.ints.shouldBeExactly
+import racingcar.domain.strategy.CanMovableStrategyMock
+import racingcar.domain.strategy.CanNotMovableStrategyMock
 
 class CarTest : DescribeSpec({
-    describe("자동차 이동 테스트") {
-        it("입력된 숫자가 4 이상이면 이동한다.") {
-            val randomNumber = 4
-            val car = Car()
+    describe("자동차 테스트") {
+        it("이름이 5자를 초과하면 IllegalArgument 예외가 throw 된다.") {
+            shouldThrow<IllegalArgumentException> {
+                Car("ppororo")
+            }
+        }
+
+        it("전진 조건을 만족하면 이동한다.") {
+            val car = Car("pobi")
 
-            car.move(randomNumber)
+            car.move(CanMovableStrategyMock())
```

Comment:
MovableStrategy의 Stub 객체를 만드는 방식도 나쁘지 않지만 `함수형 인터페이스`를 람다식으로 변경하면 더 간단하게 테스트해볼 수 있을 것 같아요. 😃

또한 Kotlin의 [Trailing Lambdas](https://kotlinlang.org/docs/lambdas.html#passing-trailing-lambdas)를 활용해서 아래와 같이 변경해볼 수 있을 것 같아요. 
```suggestion
            car.move(CanMovableStrategyMock())
```

Reply:
넵 SAM으로 변경을 하니, 따로 stub 객체를 생성하지 않고 람다로 대체가 가능하네요!! 감사합니다 😄